src/ray/protobuf/common.proto

```diff
@@ -143,3 +143,19 @@ message Task {
   TaskSpec task_spec = 1;
   TaskExecutionSpec task_execution_spec = 2;
 }
+
+// Represents a resource id.
+message ResourceId {
+  // The index of the resource (i.e., CPU #3).
+  int64 index = 1;
+  // The quantity of the resource assigned (i.e., 0.5 CPU).
+  double quantity = 2;
+}
+
+// Represents a set of resource ids.
+message ResourceMapEntry {
```

Comment:
Could also convert the normal path to use this so we can remove the flatbuffer code. Not a huge deal as that code will get removed eventually but would be nice and should be little work

Reply:
I think that will get removed as part of direct actor creation, so not doing this.